CursedWordsTGBot/run_bot.hpp

```diff
@@ -16,33 +16,33 @@
 #include "server.hpp"
 #include "worker.hpp"
 #include "signalhandler.hpp"
-
+#include <grpcpp/grpcpp.h>
 
 void run_bot(std::string token){
-
-    std::unique_ptr<TgBot::Bot> ptr_bot = std::make_unique<TgBot::Bot>(token);
-
-    Logger::getInstance().setName(ptr_bot->getApi().getMe()->username);
-    Logger::getInstance().setLevel(Logger::Levels::Debug);
-
-    std::shared_ptr<Queue<ITask>> ptr_queue = std::make_shared<Queue<ITask>> ();
-    Server server(std::move(ptr_bot), ptr_queue);
-    Worker worker(ptr_queue);
-
-    SignalHandler handler({ SIGINT, SIGTERM }, [&](){
-            static int count = 0;
-            if (!count++) {
-                server.terminate();
-                worker.terminate();
-                Logger::getInstance().logInfo(Logger::Levels::Critical, "Recieved shutdown signal. Stop polling!");
-            } else {
-                Logger::getInstance().logInfo(Logger::Levels::Fatal, "Recieved second shutdown signal. Exiting!"); 
-                std::exit(EXIT_FAILURE);
+        grpc_init();
+        std::unique_ptr<TgBot   ::Bot> ptr_bot = std::make_unique<TgBot::Bot>(token);
+
+        Logger::getInstance().setName(ptr_bot->getApi().getMe()->username);
+        Logger::getInstance().setLevel(Logger::Levels::Debug);
+
+        std::shared_ptr<Queue<ITask>> ptr_queue = std::make_shared<Queue<ITask>> ();
```

Comment:
обычно в таких ситуациях используют auto слева, чтобы код не повторять
```suggestion
        auto ptr_queue = std::make_shared<Queue<ITask>> ();
```